### PR TITLE
Added more bus api events

### PR DESCRIPTION
**Pull request by architectt1**
All events are sent when needed.

### CODE REVIEW

File: src/constants.js [hunk 1]
```diff
@@ -61,6 +61,12 @@ export const BUS_API_EVENT = {
   PROJECT_CANCELED: 'notifications.connect.project.canceled',
   PROJECT_ACTIVE: 'notifications.connect.project.active',
 
+  PROJECT_PHASE_TRANSITION_ACTIVE: 'project.phase.transition.active',
```
Comment (vikasrohit) [inline src/constants.js]: @gondzo I think we need to keep `notifications.connect` as prefix for all new events as well.

Comment (gondzo) [reply src/constants.js]: agreed, I'll push the change shortly

**Comment by vikasrohit** [comment]
@gondzo Although, [this](https://github.com/topcoder-platform/tc-project-service/blob/cbfaebf5a280bfc4dd0b721dc6ceb340198b549f/src/events/busApi.js#L369) code was not made in this PR, but it seems this condition is wrong as per inline doc and rationale of throwing the event mentioned afterwards. Could or should we fix it to `_.isEqual(originalWithouDetails, updatedWithouDetails)`?

**Comment by gondzo** [comment]
looks like a bug. agreed on the isEqual change.

**Comment by vikasrohit** [comment]
Could you please do that change in this same PR @gondzo? I haven't checked out the the architectt1's origin yet. Also, seems like their is conflict as well, which needed to be resolved before merging.

After these changes, I think we are safe to merge this PR into dev.

**Comment by vikasrohit** [comment]
@gondzo resolved conflict, merging the PR to reduce the number of things in flight.

**Pull request merged**